GM-SC-CHARTER.adoc

```diff
@@ -28,33 +33,39 @@ The TODO Group shall be composed of the General Members and a Steering Committee
  ... In each calendar year, the committee shall appoint one (1) member to fill the appointed position which will be vacated at the end of that year. The appointment shall be made during the month of October, by the current members of the commitee at that time.
  ... In even-numbered years, two (2) elected positions and one (1) appointed position shall be filled by the election and appointment processes.
  ... In odd-numbered years, three (3) elected positions and one (1) appointed position shall be filled by the election and appointment processes.
- ... If any committee member should resign or otherwise be unable to complete their term for any reason, with more than six (6) months remaining in their term, the remaining committee members shall appoint a replacement member. For election or appointment purposes, that member's term will end when the outgoing member's term would have ended.
+ ... If any committee member should resign or otherwise be unable to complete their term for any reason, with more than six (6) months remaining in their term, the remaining committee members may appoint a replacement member. For election or appointment purposes, that member's term will end when the outgoing member's term would have ended.
  ... If any committee member should resign or otherwise be unable to complete their term for any reason, with six (6) months or fewer remaining in their term, their position will be left unfilled until the regularly scheduled election or appointment process for that position.
  ... Elections shall be run using the Condorcet-IRV method through the Cornell online service (+++<u>+++http://civs.cs.cornell.edu/+++</u>+++). The methodology or service used in future elections may be changed via a five (5) member approval vote of current members of the committee at that time.
```

Comment:
```suggestion
 ... Elections shall be run using the Condorcet-IRV method through the Cornell online service (+++<u>+++http://civs.cs.cornell.edu/+++</u>+++). The methodology or service used in future elections may be changed via a five (5) member approval vote of current members of the committee at that time. TODO Steering Committee members are elected for a two year term.
```

Reply:
This addition should not be necessary because there is a bullet above stating "Committee member terms shall begin on January 1st of the year following election or appointment, and terminate on December 31st of the second year following election or appointment." If we want to state the 2-year term lengths explicitly, I'd add it before the sentence above.

Reply:
I think it never hurts to say the same thing in different places. Agree with @gkunz on where to position this sentence. With that change, LGTM, please merge.